enhancements/observability/multicluster-observability-addon-interoperability.md

```diff
@@ -0,0 +1,212 @@
+Every tool uses a different configuration API, export protocol and provides (or does 
+not provide) filtering/PII capabilities.
+
+### Prior art and user requests
+
+* Red Hat OpenShift as OpenTelemetry (OTLP) native platform: https://www.redhat.com/en/blog/red-hat-openshift-opentelemetry-otlp-native-platform
+* Export in-cluster metrics to 3rd party vendor https://github.com/openshift/cluster-monitoring-operator/issues/2000
+* Exporting metrics to Dynatrace https://issues.redhat.com/browse/OBSDA-433
+* Export all metrics to Dynatrace https://issues.redhat.com/browse/OBSDA-450
+* Customer asking to export metric to Splunk https://redhat-internal.slack.com/archives/C04TFRRKUA2/p1687853284985279
+
+### User Stories
+
+* As a fleet administrator, I want to export all telemetry signals collected by MCOA to an OTLP compatible endpoint(s).
+* As a fleet administrator, I want to filter sensitive data before it is exported to MCO telemetry store or 3rd party OTLP endpoint.
+* As a fleet administrator, I want to decide which data is exported to MCO telemetry store or 3rd party OTLP endpoint.
+
+### Goals
+
+* Use a single protocol (OTLP) for exporting all data to MCO and/or 3rd party system.
+* Provide a single configuration API on MCO CRD for exporting all telemetry data.
+* Provide unified filtering and routing capabilities for all exported telemetry data.
+
+### Non-Goals
+
+* Data visualization and querying.
+
+## Proposal
+
+The following sections describe in detail the required resources as well as the workflow to enable collection and forwarding of logs and traces on an RHACM managed fleet of OCP clusters.
+
+The following section describes how data exportation, routing and filtering is 
+configured in MCO and MCOA.
+
+### Workflow Description
+
+1. Configure OTLP endpoint in MCO (`MultiClusterObservability`) CR.
+2. The MCOA configures an additional OTLP exporter in the OpenTelemetry collector. The 
+   exporter is in the pipeline that receives all data.
+3. (optional) Filtering (e.g. for PII) can be configured in 
+   `OpenTelemetryCollector` CR manged by MCOA by [transformprocessor](https://github.
+   com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/transformprocessor/README.md).
+4. (optional) Routing can be configured in `OpenTelemetryCollector` CR managed by MCOA by
+   [routingprocessor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/routingprocessor).
+
+
+### API Extensions
+
+None - no new APIs for CRDs are introduced.
+
+### Implementation Details/Notes/Constraints [optional]
+
+#### General configuration and fleet-wide stanzas
+
+To support above workflow MCOA deploys additional collector which forwards all data to 
+MCO telemetry store and/or 3rd party OTLP endpoint.
+- 
+- An `OpenTelemetryCollector` resource that describes the trace exporters to be used for traces collection and forwarding on all managed clusters.
```

Comment:
From what I see above, this is not just for traces. Should `traces collection` be replaced by `telemetry data collection`?

Reply:
there was a typo, otelcol is for all signals